pages/index.html

```diff
@@ -0,0 +1,135 @@
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
+  <script src="https://code.jquery.com/jquery-3.3.1.min.js"></script>
+  <link rel="stylesheet" href="../css/style.css">
+  <!-- DRIVEBACK SYNC EXPERIMENTS SNIPPET -->
+  <style>
+    .dbex-content-hide {
+      opacity: 0 !important
+    }
+  </style>
+  <script type="text/javascript">
+    window.DBEX_HIDE_CONTENT_STRATEGY = 'sync'; // async/sync (default: sync)
+    window.DBEX_HIDE_CONTENT_TIMEOUT = 1000; // minimum 1000/4000 for sync/async
+  </script>
+  <script type="text/javascript" src="../js/dbex-sync.min.js"></script>
+  <!-- // DRIVEBACK SYNC EXPERIMENTS SNIPPET -->
+</head>
+
+<body>
+  <header>
+    <div class="container">
+      <div class="header-content">
+        <img src="../img/logo/db@2x.png" alt="logo">
+        <h1>Driveback sync experiments</h1>
+      </div>
+    </div>
+  </header>
+  <main>
+    <section class="first-block">
+      <div class="container">
+        <div class="main-content">
+          <h1 id="main_title">Сайт #1 (Контрольная группа | 20% трафика)</h1>
+          <img id="main_banner" src="../img/banner1.jpg" />
```

Comment:
Header design is awful. All banners should be changed and should fit design. Previous banners were just examples, they even have different size. Please, consult with designer.